test/fixtures/ecdsa.json

```diff
@@ -6,70 +6,49 @@
         "k": "ec633bd56a5774a0940cb97e27a9e4e51dc94af737596a0c5cbb3d30332d92a5",
         "message": "Everything should be made as simple as possible, but not simpler.",
         "i": 0,
-        "signature": {
-          "r": "23362334225185207751494092901091441011938859014081160902781146257181456271561",
-          "s": "50433721247292933944369538617440297985091596895097604618403996029256432099938"
```

Comment:
These were all in decimal,  which is just confusing IMHO.  I switched them to DER.